catalog/designers/christiannaths/info.pb

```diff
@@ -0,0 +1,5 @@
+designer: "Christian Naths"
+link: ""
+avatar {
+  file_name: "Christiannaths.png"
```

Comment:
we don't want uppercases in files name, @emmamarichal can you update the actual png and this link?